Tests/SHLLMTests/Models/Qwen2_5-7BTests.swift

```diff
@@ -19,3 +19,47 @@ func canLoadAndQueryQwen2_5__7B() async throws {
     Swift.print(result)
     #expect(!result.isEmpty)
 }
+
+@Test
+func canHelpMeFetchTheWeatherWithQwen2_5__7B() async throws {
+    guard let llm = try await Qwen2_5__7B.tests else { return }
+
+    let tool1: WeatherTool = try await llm.request(
+        tools: Tools([weatherToolFunction]),
+        messages: [
+            [
+                "role": "system",
+                "content": "You are a weather fetching assistant. Your only purpose is to fetch weather data.",
+            ],
+            ["role": "user", "content": "What is weather in Paris like?"],
+        ]
+    )
+
+    let expectedTool1 = WeatherTool.getCurrentWeather(.init(
+        location: "Paris, France",
+        unit: .celsius
+    ))
+
+    print("\(#function) 1:", tool1)
+    #expect(tool1 == expectedTool1)
+
+    let tool2: WeatherTool = try await llm.request(
+        tools: Tools([weatherToolFunction]),
+        messages: [
+            [
+                "role": "system",
+                "content": "You are weather fetching assistant. Your only purpose is to fetch weather data.",
+            ],
+            ["role": "system", "content": "The user prefers F°."],
```

Comment:
@myobie I swapped Fahrenheit for Celsius here just to see if **7B** listens to us. It seems to most of the time.

Reply:
Nice.